inject/chrome.js

```diff
@@ -20,7 +20,28 @@ function randomString () {
     return num.toString().replace('0.', '')
 }
 
+/**
+ * @see ../src/trackers.js
+ */
+function isTrackerOrigin (trackerLookup) {
+    const originHostname = document.location.hostname
+    const parts = originHostname.split('.').reverse()
+    let node = trackerLookup
+    for (const sub of parts) {
+        if (node[sub] === '1') {
+            return true
+        } else if (node[sub]) {
+            node = node[sub]
+        } else {
+            return false
+        }
+    }
+    return false
+}
+
 function init () {
+    const documentOriginIsTracker = isTrackerOrigin($TRACKER_LOOKUP$)
+    const bundledConfig = $BUNDLED_CONFIG$
```

Comment:
The extension config is pretty massive, and most of it seems irrelevant to C-S-S. Should we prune it somehow before injection?

Reply:
Yeah, we'll do this on the extension side. I'll add it to that PR.